components/NavBar/NavBar.tsx

```diff
@@ -61,22 +39,36 @@ const HeaderContentContainer = styled(Container)`
     margin-right: 5rem;
   }
 `
+const TopBarContainer = styled(TopBar)`
+  width: 100%;
+  padding-bottom: 4rem;  
 
+`
 const LogoContainer = styled.div`
   width: fit-content;
   white-space: nowrap;
 `
 
 const ActionsContainer = styled(TopBar.Actions)`
+  width: 100%;
+  background-color: ${tokens.colors.ui.background__default.hex};
   display: flex;
+  justify-content: flex-end;
+
+`
+
+const SearchContainer = styled(TopBar.CustomContent)`
```

Comment:
Generally speaking, I think we are overdoing the styling here. We shouldn't need so much custom styles on the top of EDS. I think these custom styles can be removed all together and the same with the search form styles. If I do that, it will look like this and that's more aligned with EDS and a lot less styles
![Skjermbilde 2022-08-19 kl  13 09 52](https://user-images.githubusercontent.com/1568252/185606443-39c68542-a4e3-4aa8-84fe-20d9f0eab306.png)